addons/mail/security/mail_security.xml

```diff
@@ -96,6 +96,29 @@
             <field name="perm_unlink" eval="False"/>
         </record>
 
+        <record id="ir_rule_discuss_channel_member_is_channel_owner_write_unlink" model="ir.rule">
+            <field name="name">discuss.channel.member: channel owner/admins can write/unlink entries of their channel</field>
```

Comment:
Write should not be allowed as there are many fields on member that should stay private to the member itself.